electron-app/src/components/settings/api-keys/ExchangeSettings.vue

```diff
@@ -99,16 +105,22 @@ import BaseExternalLink from '@/components/base/BaseExternalLink.vue';
 import ConfirmDialog from '@/components/dialogs/ConfirmDialog.vue';
 import MessageDialog from '@/components/dialogs/MessageDialog.vue';
 import ExchangeBadge from '@/components/ExchangeBadge.vue';
+import { convertToGeneralSettings } from '@/data/converters';
 import { exchanges } from '@/data/defaults';
 import { Message } from '@/store/store';
+import { GeneralSettings } from '@/typing/types';
 
-const { mapState } = createNamespacedHelpers('balances');
+const { mapState } = createNamespacedHelpers('session');
 
 @Component({
   components: { ConfirmDialog, MessageDialog, ExchangeBadge, BaseExternalLink },
-  computed: mapState(['connectedExchanges'])
+  computed: {
+    ...mapState(['connectedExchanges']),
+    ...mapState(['settings'])
```

Comment:
then here you can replace with the `...mapGetters(['krakenAccountType'])`

Reply:
done